README.md

```diff
@@ -38,6 +38,7 @@ npx @ciena-org/ember-codemods $TRANSFORM path/of/files/ or/some**/*glob.js
 | Transform | Description | Possible issues |
 | --------- | ----------- | --------------- |
 | [revert-computed-macro](./src/transforms/revert-computed-macro/)| Replace `computed` from `ember-macro-helpers` with `@ember/object` instead. | <ul><li>Will add variable even if not used from function before. Can just remove as need be (or PR fix), was not a common occurence in my code to have unused dependency</li></ul> |
+| [ember-getWithDefault-to-lodash-get](./src/transforms/ember-getWithDefault-to-lodash-get/) | Replace ember `getWithDefault` with lodash `get` instead | |
```

Comment:
This entry isn't necessary since this transform aligns with a deprecation. This table is more to document any transforms that we write that aren't really related to deprecations
```suggestion
```

Reply:
Done